load_tests/daemonset/cloudwatch.yaml

```diff
@@ -0,0 +1,248 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: load-test-fluent-bit-eks
+  namespace: load-test-fluent-bit-eks-ns
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: load-test-fluent-bit-eks-role
+rules:
+  - apiGroups: [""]
+    resources:
+      - namespaces
+      - pods
+    verbs: ["get", "list", "watch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: load-test-fluent-bit-eks-role-binding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: load-test-fluent-bit-eks-role
+subjects:
+  - kind: ServiceAccount
+    name: load-test-fluent-bit-eks
+    namespace: load-test-fluent-bit-eks-ns
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cloudwatch-$THROUGHPUT-config
+  namespace: load-test-fluent-bit-eks-ns
+  labels:
+    k8s-app: load-test-fluent-bit-eks
+data:
+  fluent-bit.conf: |
+    [SERVICE]
+        Flush         1
+        Grace         30
+        Log_Level     info
+        Daemon        off
+        Parsers_File  parsers.conf
+        HTTP_Server   On
+        HTTP_Listen   0.0.0.0
+        HTTP_Port     2020
+        
+    @INCLUDE application-log.conf
+  
+  application-log.conf: |
+    [INPUT]
+        Name                tail
+        Tag                 kube.*
+        Path                /var/log/containers/ds-cloudwatch-$THROUGHPUT*
+        Docker_Mode         On
+        Parser              docker
+        DB                  /var/fluent-bit/cloudwatch/$THROUGHPUT/flb_log.db
+        DB.locking          true
+        Mem_Buf_Limit       5GB
+        buffer_chunk_size   80MB
+        buffer_max_size     5GB
+        Skip_Long_Lines     On
+        Skip_Empty_Lines    On
+        Refresh_Interval    10
+        Read_from_Head      On
+        Rotate_Wait         600
+
+    [FILTER]
+        Name                kubernetes
+        Match               kube.*
+        Kube_URL            https://kubernetes.default.svc:443
+        Kube_Tag_Prefix     kube.var.log.containers
+        Merge_Log           On
+        Merge_Log_Key       log_processed
+        K8S-Logging.Parser  On
+        K8S-Logging.Exclude Off
+        Labels              Off
+        Annotations         Off
+
+    [OUTPUT]
+        Name                cloudwatch_logs
+        Match               kube.*
+        region              us-west-2
+        log_group_name      $CW_LOG_GROUP_NAME
+        log_stream_prefix   load-test-fluent-bit-
+        auto_create_group   false
+        log_key             log
+        workers             1
+        auto_retry_requests true
```

Comment:
we should configure a retry_limit on the outputs right?

Reply:
good catch. I will add it. And also, I am keeping editing this config to improve the result now so it might still be updated based on this version